Makefile

```diff
@@ -41,7 +41,7 @@ battletest: ## Run randomized, racing, code coveraged, tests
 
 e2etests: ## Run the e2e suite against your local cluster
 	go clean -testcache
-	go test -p 1 -timeout 60m -v ./test/suites/... -run=${TEST_FILTER}
+	go test -p 1 -timeout 180m -v ./test/suites/... -run=${TEST_FILTER}
```

Comment:
Wanna make it 4 hours to match the interval?

Reply:
3 hrs is just for the tests. That gives it 1 hour to setup and tear down.